src/error-page.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { useRouteError } from "react-router-dom";
+
+export default function ErrorPage() {
+  const error = useRouteError() as unknown;
+  console.error(error);
+
+  return (
+    <div id="error-page">
+      <h1>Oops!</h1>
+      <p>Sorry, an unexpected error has occurred.</p>
+      <p>
+        <i>Wrong URL address</i>
```

Comment:
How could you know that in advance? There might occur any type of error, not only "Wrong URL address". So, please, remove lines 12-14.